x-pack/plugin/sql/qa/server/src/main/resources/select.sql-spec

```diff
@@ -54,6 +54,10 @@ sameConstantsWithLimitV2
 SELECT 5, 3, 3 FROM "test_emp" LIMIT 5;
 sameConstantsWithLimitV3
 SELECT 3, 5, 3, 3 FROM "test_emp" LIMIT 5;
+constantWithWhere
+SELECT 1 FROM "test_emp" WHERE gender = 'F';
+constantWithWhereOrderAndLimit
+SELECT 1 FROM "test_emp" WHERE gender = 'F' ORDER BY emp_no LIMIT 5;
```

Comment:
Please add more tests to test different varieties of tested plan:
`WHERE a and b`
`WHERE a and function(b)`
`WHERE a and b ORDER BY c`